### ISSUE TITLE
Code review on pull request and repository push

### ISSUE THREAD

**Issue by hadar-cpp-check** [probot/ideas#108]
Hi, 
I'm new to probot and github marketplace,
I'm trying to create an app that will run some code review once a user try to push a commit.

then my code will decide either to pass or fail the push an present the user some error messages incase of failure.

can you please advise what will be the best way to build such a thing?

**Comment by dudo** [comment]
If you're reviewing code from the repository, I'd suggest building a GitHub Action, instead.

**Comment by MahmoudMabrok** [comment]
I agree with @dudo , github actions is enough, no need to build app for this task. 

cc: @hadar-cpp-check
